### PR TITLE
Improved documentation to NSViewExtensions

### PR DESCRIPTION
**Pull request by herlandro**
🚀

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 5.6.
- [x] New extensions support iOS 12.0+ / tvOS 12.0+ / macOS 10.13+ / watchOS 4.0+, or use `@available` if not.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [ ] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Sources/SwifterSwift/AppKit/NSViewExtensions.swift [hunk 1]
```diff
@@ -6,7 +6,13 @@ import AppKit
 // MARK: - Properties
 
 public extension NSView {
-    /// SwifterSwift: Border color of view; also inspectable from Storyboard.
+    /// SwifterSwift: Border color of the view; also inspectable from Storyboard.
+    ///
+    ///    ```swift
+    ///    view.borderColor = NSColor.red
```
Comment (guykogus) [inline Sources/SwifterSwift/AppKit/NSViewExtensions.swift:12]: Can't you just do this?
```suggestion
    ///    view.borderColor = .red
```

File: Sources/SwifterSwift/AppKit/NSViewExtensions.swift [hunk 2]
```diff
@@ -6,7 +6,13 @@ import AppKit
...
+    ///    ```
+    ///
+    /// - Returns: The border color of the view as `NSColor`.
```
Comment (guykogus) [inline Sources/SwifterSwift/AppKit/NSViewExtensions.swift:15]: You don't need a `Returns` for a computed property.

**Review by guykogus — CHANGES_REQUESTED** [review]
I don't think we really need an improvement for this documentation. It's fairly straightforward...

**Comment by herlandro** [comment]
> I don't think we really need an improvement for this documentation. It's fairly straightforward...

Ok, No problem. There is some place on your project that you need help and that I can help?

**Comment by guykogus** [comment]
@herlandro sorry for the very late reply. If you want to help please check the [Issues](https://github.com/SwifterSwift/SwifterSwift/issues) and see what you can take. Thanks :)

**Pull request closed without merging**